chainer_compiler/chainer_compiler.py

```diff
@@ -203,6 +203,34 @@ def _run_translator(translator, mc, inputs):
     return graph
 
 
+def _resolve_name_correspondence(mc, params, name, param_values):
+    for running_mean_name in ('avg_mean', 'running_mean'):
```

Comment:
Any reason we need to support both `avg_mean` and `running_mean`? Also, could you put a link to onnx-chainer to mention this code assumes the naming convention of onnx-chainer?